sailthru/sailthru_client.py

```diff
@@ -333,10 +328,10 @@ def delete_template(self, template_name):
         data = {'template': template_name}
         return self.api_delete('template', data)
 
-    def save_template(self, template, template_fields=None):
-        template_fields = template_fields or {}
-        data = template_fields.copy()
-        data['template'] = template
+    def save_template(self, template, **template_fields):
```

Comment:
This could be a breaking change if a client wrote `save_template(name, template_fields=my_dict)`.
This would create a payload o `'{'template': 'my template', 'template_fields': {'content_html': ...}}`

Despite the potential to break client code, this is the proper idiom to use for optional fields.